packages/react-keytips/src/docs/Spec.md

```diff
@@ -0,0 +1,191 @@
+# @fluentui-contrib/react-keytips Spec
+
+## Background
+
+A `Keytip` is a small Tooltip near a component that indicates a key sequence that will trigger that component. There are
+not to be confused with keyboard shortcuts, they are instead key sequence to traverse through levels of UI components.
+
+## Prior Art
+
+- [Convergence Epic](https://github.com/microsoft/fluentui/issues/26420)
```

Comment:
This link leads to Drawer component. I don't see Epic for Keytips. Not sure if we need it for contrib components

Reply:
thanks! I will keep it in case if somebody will later would like to look up for epic. ;)